test/test_cuda/test_mix_bits.py

```diff
@@ -58,17 +57,16 @@ def test_mixed_gptqmodel(self):
         quantized_model_path = "./saved"
         autoround.quantize_and_save(output_dir=quantized_model_path, format="auto_gptq")
         from gptqmodel import GPTQModel
-
         model = GPTQModel.load(quantized_model_path)
-        assert model.model.model.decoder.layers[0].self_attn.k_proj.bits == 8
-        assert model.model.model.decoder.layers[0].self_attn.q_proj.bits == 4
-        result = model.generate("Uncovering deep insights begins with")[0]  # tokens
-        assert "!!!" not in model.tokenizer.decode(result)  # string output
+        assert (model.model.model.decoder.layers[0].self_attn.k_proj.bits == 8)
+        assert (model.model.model.decoder.layers[0].self_attn.q_proj.bits == 4)
```

Comment:
test the regex , full name, part name and make sure the model could inference correctly.